unified-codes/apps/web/src/components/molecules/ExplorerTable.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+
+import { EntityTable } from '@unified-codes/ui';
+
+import ExplorerTableHeader from './ExplorerTableHeader';
+import ExplorerTableRows from './ExplorerTableRows';
+import ExplorerTablePagination from './ExplorerTablePagination';
+
+export const ExplorerTable = () => <EntityTable header={<ExplorerTableHeader/>} rows={<ExplorerTableRows/>} pagination={<ExplorerTablePagination/>}/>
```

Comment:
Example of composing connected components together. Have to be careful with this as it works on the assumption that these components are all "dumb" and get their state directly from the app store.

Reply:
You're 100% right that it's only a convenience wrapper. It's really just a placeholder to maintain a consistent API!

e.g. `EntityTableHeader`, `EntityTableRow`, `EntityTableX`, etc. etc. (I also think we should do one with `EntityTablePagination`, which would be similarly pointless, but maintain a clear API).

Does that make sense?

Reply:
yep, makes sense.